plugin-server/src/main/graphile-worker/graphile-worker.ts

```diff
@@ -94,10 +94,9 @@ export class GraphileWorker {
         await instrument(
             this.hub.statsd,
             {
-                metricName: 'job_queues_enqueue',
+                metricName: 'job_queues_enqueue_${jobName}',
                 key: instrumentationContext?.key ?? '?',
                 tag: instrumentationContext?.tag ?? '?',
-                tags: { jobName, type: jobType },
```

Comment:
This was the only place using `tags` and all we need is the job name. This makes the Prometheus metric labels much cleaner.